engines/stark/stark.cpp

```diff
@@ -336,7 +336,8 @@ Common::Error StarkEngine::loadGameState(int slot) {
 
 bool StarkEngine::canSaveGameStateCurrently() {
 	// Disallow saving when there is no level loaded or when a script is running
-	return _global->getLevel() && _userInterface->isInteractive();
+	// or when the save & load menu is currently displayed
+	return _global->getLevel() && _userInterface->isInteractive() && !_userInterface->isInSaveLoadMenuScreen();
 }
```

Comment:
I used to try an implementation that allows saving through GMM while the save & load menu is displayed, but it is kind of complex since the menu needs to be notified when a save slot is updated and update the thumbnail of the corresponding widget. Maybe just disable the saving is a better approach?